tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -508,6 +512,13 @@ def Validate(self, entity: EntityInstance) -> bool:
     Returns:
       True if the entity is valid
     """
+
+    if IsEntityIdPresent(entity):
```

Comment:
You could also just do if entity.entity_id:

I think that'll save you some lines.